lib/Parse/ParseDecl.cpp

```diff
@@ -3324,7 +3336,7 @@ parseIdentifierDeclName(Parser &P, Identifier &Result, SourceLoc &Loc,
                         StringRef DeclKindName, tok ResyncT1, tok ResyncT2,
                         tok ResyncT3, tok ResyncT4,
                         TokenProperty ResyncP1) {
-  if (P.Tok.is(tok::identifier)) {
+  if (P.Tok.is(tok::identifier) || P.Tok.is(tok::kw_call)) {
```

Comment:
Added to support parsing `call` as an identifier in non-declaration contexts.
This enables `func call` and `call(...)` (apply expressions).